Sources/muterCore/fileOperations.swift

```diff
@@ -22,3 +25,8 @@ func createLoggingDirectory(in directory: String,
     try! fileManager.createDirectory(atPath: loggingDirectory, withIntermediateDirectories: true, attributes: nil)
     return loggingDirectory
 }
+
+public struct SourceCodeInfo {
```

Comment:
1. Put this in its own file?
1. Is it right that the type is `public` and its properties are `internal`?